ios/BT/API/Requests/ExposureDetectionSummaryRequests.swift

```diff
@@ -0,0 +1,53 @@
+import Alamofire
+import ExposureNotification
+
+enum ExposureDetectionSummaryListRequest: APIRequest {
+
+  typealias ResponseType = ScoringServerResponse
+
+  case post(ExposureDetectionSummary, [ExposureDetectionSummary])
+
+  var method: HTTPMethod {
+    switch self {
+    case .post:
+      return .post
+    }
+  }
+
+  var path: String {
+    switch self {
+    case .post:
+      return ""
+    }
+  }
+
+  var parameters: Parameters? {
+    switch self {
+    case .post(let newExposureDetectionSummary,
+              let storedExposureDetectionSummaries):
+      return [
+        "new_exposure_summary": [
+          newExposureDetectionSummary.asJsonBody,
+        ],
+        "unused_exposure_summaries": storedExposureDetectionSummaries.map { $0.asJsonBody }
+      ]
+    }
+  }
+}
+
+private extension ExposureDetectionSummary {
+
+  var asJsonBody: JSONObject {
+    return [
+        "date_received": startOfDateReceived,
+        "timezone_offset": timezoneOffset,
+        "attenuation_durations_seconds": [],
```

Comment:
This needs to be the low/medium/high buckets instead right?

Reply:
good catch, this is out of date - incorrect parameters and also I need to use camelcase